webapp/client/src/variables.css

```diff
@@ -0,0 +1,90 @@
+:root {
+  /* Colors */
+  --text-color: #0b0c0c;
+  --secondary-text-color: #545454;
+  --inverse-text-color: #ffffff;
+
+  --border-color: #bababa;
+  --hover-border-color: #003b52;
+  --active-outline-color: #cdc0b1;
+
+  --link-color: #005576;
+  --link-hover-color: #003b52;
+  --link-visited-color: #100052;
+  --link-active-color: #0b0c0c;
+  --link-underline-color: rgba(10, 10, 25, 0.2); /* original #0A0A19, 20%; */
+
+  --icon-default-color: #e3d9ce;
+  --icon-hover-color: #cdc0b1;
+
+  --focus-color: #ffc979;
+  --focus-text-color: #0b0c0c;
+  --focus-border-color: #0b0c0c;
+
+  --body-color: #fdf8f4;
+  --bg-highlight-color: #eee8e1;
+  --bg-white: #ffffff;
+
+  --success-color: #005c45;
+  --error-color: #c0003c;
+  --error-bg-color: #f9e5eb;
+
+  /* Font Size */
+  --text-sm: 0.785rem;
+  --text-s: 0.9rem;
+  --text-base: 1rem;
+  --text-medium: 1.125rem;
+  --text-xl: 1.375rem;
+  --text-2xl: 1.625rem;
+  --text-3xl: 1.875rem;
+  --text-4xl: 2.25rem;
+
+  /* Font Weight */
+  --font-normal: 400;
+  --font-bold: 700;
+
+  /* Letter Spacing */
+  --tracking-tight: -0.02em;
+  --tracking-normal: 0;
+  --tracking-wide: 0.02em;
+  --tracking-extra-wide: 0.07em;
+
+  /* Line Height */
+
+  --lineheight-none: 1;
+  --lineheight-xs: 1.1;
+  --lineheight-s: 1.2;
+  --lineheight-m: 1.3;
+  --lineheight-default: 1.45;
+  --lineheight-l: 1.6;
+
+  /* Spacing */
+
+  --spacing-01: 0.375rem;
+  --spacing-02: 0.625rem;
+  --spacing-03: 1rem;
+  --spacing-04: 1.25rem;
+  --spacing-05: 1.5rem;
+  --spacing-06: 1.875rem;
+  --spacing-07: 2.5rem;
+  --spacing-08: 3.125rem;
+  --spacing-09: 3.75rem;
+  --spacing-10: 5.625rem;
+  --spacing-11: 7.5rem;
+
+  /* Input Field Widths */
+  --input-width-2: 3.5rem;
+  --input-width-3: 3.8rem;
+  --input-width-4: 5rem;
+  --input-width-5: 5.8rem;
+  --input-width-6: 7rem;
+  --input-width-10: 16.5rem;
+  --input-width-25: 23.5rem;
+
+  /* Specifics */
+
+  --sidebar-width: 220px;
+  --main-max-width: 1000px;
+  --back-link-size: 1.8em;
+  --header-navigation-height: var(--spacing-09);
```

Comment:
Probably, we could move this somewhere else.

Reply:
Yeah, definitely. As mentioned in another thread, I'd like to only change what's necessary at this point and do cleanup / refactorings later.